docs/content/quickstart/parameters.md

```diff
@@ -0,0 +1,147 @@
+---
+title: "QuickStart: Parameters"
+descriptions: Learn how to use a bundle with parameters
+layout: single
+---
+
+Now that you know how to install a bundle, let's look at how to specify parameters to customize how that bundle is installed.
+Bundles can define parameters to allow the end-user to tweak how the bundle is configured.
+A parameter can be a string, integer, boolean or even a json object.
+Some examples of how parameters can be used in a bundle are:
+
+* Log Level: Default the log level for an application to info. At any time a user can upgrade the bundle to change that parameter to a different value.
+* Deployment Region: Let the user specify which region, such as eastus-1, where the application should be deployed.
+* Helm Release Name: A bundle that uses Helm often will define a parameter that allows the user to set the release name for the Helm release.
+
+For optional parameters, bundles set a default value that is used when the user does not specify a value for the parameter.
+
+Let's look at a bundle with parameters:
+
+```console
+$ porter explain --reference getporter/hello-llama:v0.1.1
+Name: hello-llama
+Description: An example Porter bundle with parameters
+Version: 0.1.0
+Porter Version: v0.38.1-32-gb76f5c1c
+
+No credentials defined
+
+Parameters:
+Name   Description                           Type     Default   Required   Applies To
+name   Name of to whom we should say hello   string   llama     false      All Actions
+
+No outputs defined
+
+No custom actions defined
+
+No dependencies defined
+```
+
+The output tells us that the bundle has one optional string parameter, name, which defaults to "llama".
+The name parameter applies to "All Actions" meaning that this parameter can be specified with every action that the bundle defines.
+Since it says that no custom actions are defined, the bundle only supports the built-in actions of install, upgrade, and uninstall.
+
+## Use the default parameter values
+
+
+First install the bundle and do not specify any parameters so that you can observe how the bundle installs without any customization.
+
+```console
+$ porter install hello-llama --reference getporter/hello-llama:v0.1.1
+installing hello-llama...
+executing install action from hello-llama (installation: hello-llama)
+Hello, llama
+execution completed successfully!
+```
+
+The bundle printed "Hello, llama" using the default value for the name parameter.
+
+## Specify a parameter with a flag
+Next upgrade the installation and change the name parameter to another value.
+Parameters are specified with the \--param flag: 
+
+```console
+$ porter upgrade hello-llama --param name=Porter
```

Comment:
Maybe use a different name than Porter so it isn't confusing to new folks. (Michele?)